notesserver/views.py

```diff
@@ -38,12 +40,12 @@ def heartbeat(request):  # pylint: disable=unused-argument
     try:
         db_status()
     except Exception:
-        return Response({"OK": False, "check": "db"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+        return JsonResponse({"OK": False, "check": "db"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
     if not settings.ES_DISABLED and not get_es().ping():
-        return Response({"OK": False, "check": "es"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+        return JsonResponse({"OK": False, "check": "es"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
-    return Response({"OK": True})
+    return JsonResponse({"OK": True}, status=status.HTTP_200_OK)
```

Comment:
Yes, `/heartbeat` sends HTTP response by default, so we have been asked to change this as well - See [TNL-5244](https://openedx.atlassian.net/browse/TNL-5244)